5_data_munge/src/munge_data_files.R

```diff
@@ -179,3 +179,63 @@ summarize_all_dat <- function(in_ind, out_file) {
 
   readr::write_csv(summary, out_file)
 }
+
+clean_sites <- function(in_ind, out_ind) {
+
+  proj_albers <- "+proj=laea +lat_0=45 +lon_0=-100 +x_0=0 +y_0=0 +a=6370997 +b=6370997 +units=m +no_defs"
+
+  states <- st_as_sf(map("state", plot = FALSE, fill = TRUE)) %>%
```

Comment:
I wanted to explore this line and happened to load **tidyverse** after **maps**. The result was an error:
```
> x <- map("state", plot = FALSE, fill = TRUE)
Error in as_mapper(.f, ...) : argument ".f" is missing, with no default

> x <- maps::map("state", plot = FALSE, fill = TRUE)

> environment(map)
<environment: namespace:purrr>
```

So even though the pipeline probably works fine from a fresh build, it'd probably be smart to qualify this call as `maps::map` in case we later add tidyverse or purrr with precedence over sf.